db/config.cc

```diff
@@ -826,6 +826,9 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , enable_sstables_mc_format(this, "enable_sstables_mc_format", value_status::Unused, true, "Enable SSTables 'mc' format to be used as the default file format.  Deprecated, please use \"sstable_format\" instead.")
     , enable_sstables_md_format(this, "enable_sstables_md_format", value_status::Unused, true, "Enable SSTables 'md' format to be used as the default file format.  Deprecated, please use \"sstable_format\" instead.")
     , sstable_format(this, "sstable_format", value_status::Used, "me", "Default sstable file format", {"md", "me"})
+    , enable_uuid_sstable_identifiers(this,
+            "uuid_sstable_identifiers_enabled", value_status::Used, false, "If set to true, each newly created sstable will have a UUID "
```

Comment:
Name should match variable name.

Reply:
okay. will incorporate this suggestion in another PR.

Reply:
When I see a field in the `scylla.yaml`, I come here and search for that exact name and expect to find the member variable holding that value. Not finding it like that would be terribly confusing and annoying too. I think I'm not alone doing this.

Reply:
yeah. makes sense to me. this saves a hop in searching. and developer's time is precious.